index.html

```diff
@@ -0,0 +1,430 @@
+    </head>
+    <body>
+
+      <header>
+<!-- Menu Bar -->
+        <nav class="mobile-menu">
+          <label for="show-menu" class="show-menu">
+             <span class="icon-menu"></span>
+          </label>
+         <input type="checkbox" id="show-menu">
+          <ul class="menu">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="index.html#services">Services</a></li>
+            <li><a href="index.html#portfolio">Portfolio</a></li>
+            <li><a href="index.html#about">About</a></li>
+            <li><a href="index.html#team">Team</a></li>
+          </ul>
+        </nav>
+      </header>
+
+      <!-- Hi There Avengers first section -->
+      <section class="p-home">
+        <div class="l-home-spacer"> </div>
+        <div class="l-icon-outer">
+          <div class="l-droplet-icon">
+            <div class="l-droplet-icon-interior"></div>
+          </div>
+        </div>
+        <div class="l-home-text">
+          <h1>Hi there! We're Avengers, <strong>creative design agency</strong>. We build <strong><u>awesome</u></strong> mobile and websites</h1>
+        </div>
+        <div class="l-home-spacer"> </div>
+      </section>
+<!-- Services section -->
+      <section class="p-services" id="services">
+          <div class="p-divet"></div>
+
+        <section class="l-title">
+          <div class="p-title-line"></div>
+          <h2 class="p-title">Services</h2>
+          <div class="p-title-line"></div>
+        </section>
+
+        <p class="l-upper-rows">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+        <p class="l-upper-columns">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+        <p class="l-upper-columns">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+
+        <section class="l-services-list">
+            <figure class="l-services-item">
+            <h3>UI & UX DESIGN</h3>
+              <div class="l-icon-outer">
+                <span class="icon-paint-format"></span>
+              </div>
+              <figcaption>There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</figcaption>
+            </figure>
+            <!-- <div class="l-home-spacer"> </div> -->
+            <figure class="l-services-item">
+            <h3>LABORATORY</h3>
```

Comment:
I see what you are doing here. You shouldn't use the same heading element multiple times within a component. I would suggest making this parent element an `article` and then each sub-area a `section` element. Then using a `h1` again is legal.